awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsTracerProviderFactory.java

```diff
@@ -15,13 +15,12 @@
 
 package software.amazon.opentelemetry.javaagent.providers;
 
-import io.opentelemetry.api.OpenTelemetry;
-import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.api.trace.TracerProvider;
 import io.opentelemetry.sdk.extension.aws.trace.AwsXrayIdGenerator;
 import io.opentelemetry.sdk.trace.SdkTracerProvider;
-import io.opentelemetry.spi.OpenTelemetryFactory;
+import io.opentelemetry.spi.trace.TracerProviderFactory;
 
-public class AwsOpenTelemetryFactory implements OpenTelemetryFactory {
+public class AwsTracerProviderFactory implements TracerProviderFactory {
```

Comment:
Unnecessary changes, these will change again in 0.15.0 to `implements TracerProviderConfigurer`